pkg/config/derived-service-cert.go

```diff
@@ -60,6 +63,11 @@ func (idCfg *IdentityConfig) derivedServiceCertConfig() error {
 		domainName := extutil.NamespaceToDomain(idCfg.Namespace, idCfg.athenzPrefix, idCfg.athenzDomain, idCfg.athenzSuffix)
 		domainDNSPart := extutil.DomainToDNSPart(domainName)
 
+		// set instance certificate specific attributes
```

Comment:
Use the anonymous function will make it look clean (I dont think the performance varies that much.)

Reply:
The anonymous function will have side-effect to update the source struct .

what we do here